citesphere/src/main/java/edu/asu/diging/citesphere/web/user/DeleteCitationController.java

```diff
@@ -0,0 +1,56 @@
+package edu.asu.diging.citesphere.web.user;
+
+import java.util.List;
+import java.util.Properties;
+
+import org.javers.common.collections.Arrays;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.social.zotero.exception.ZoteroConnectionException;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import edu.asu.diging.citesphere.core.exceptions.CannotFindCitationException;
+import edu.asu.diging.citesphere.core.exceptions.GroupDoesNotExistException;
+import edu.asu.diging.citesphere.core.exceptions.ZoteroHttpStatusException;
+import edu.asu.diging.citesphere.core.exceptions.ZoteroItemCreationFailedException;
+import edu.asu.diging.citesphere.core.service.ICitationManager;
+
+@Controller
+public class DeleteCitationController {
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    @Value("${_creation_default_item_type}")
+    private String defaultItemType;
+
+    @Autowired
+    @Qualifier("creatorsFile")
+    private Properties properties;
+
+    @Autowired
+    private ICitationManager citationManager;
+
+
+    @RequestMapping(value = "/auth/group/{groupId}/delete", method = RequestMethod.POST)
+    public String delete(Authentication authentication,@PathVariable("groupId") String groupId, @RequestParam(value="citationList", required=false) String citationList)
```

Comment:
I think you can have spring parse the list of ids directly into a list or array by not using `String citationList` (but List or array).